README.md

```diff
@@ -160,6 +160,30 @@ async function test() {
   const payload = await verifyWithPromise(token)
   // => { a: 1, b: 2, c: 3, iat: 1579521212 }
 }
+
```

Comment:
This should probably be a seprate code block.

Reply:
Yep! Agreed